.gitlab-ci.yml

```diff
@@ -29,7 +29,7 @@ build:
     - echo "Ignore warning! Cannot perform an interactive login from a non TTY device"
     - docker push $IMAGE_ORG/$IMAGE_NAME:$CI_COMMIT_SHA
 
-lint:
+.lint:
```

Comment:
We shouldn't disable linting in this PR

Reply:
No, of course. But I'm hammering this pipeline trying to debug the failures and this saves a few precious seconds each time.